lib/carrierwave/uploader/url.rb

```diff
@@ -15,8 +15,8 @@ module Url
       # [String] the location where this file is accessible via a url
       #
       def url(options = {})
-        if file.respond_to?(:url) and not (tmp_url = file.url).blank?
-          file.method(:url).arity == 0 ? tmp_url : file.url(options)
+        if file.respond_to?(:url) && !file.url.blank?
+          file.method(:url).arity.zero? ? file.url : file.url(options)
```

Comment:
`tmp_url` should not be removed, because `#url` might be costly operation when using fog.

Reply:
I didn't know ```#url``` is costly when using fog.
I added ```tmp_url``` again.

Thank you.